utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java

```diff
@@ -137,4 +137,27 @@ public void resetState() {
     public void sleepForJenkinsDelay(int secs) {
         TestUtils.sleepForJenkinsDelayIfHigher(secs);
     }
+
+    @Then("^wait for Todo to become ready")
```

Comment:
is the $ end of line left out intentionally from the regex?

Reply:
it is not. I just forgot about it. I will update it.